src/Donations/resources/admin/components/DonationDetailsPage/index.tsx

```diff
@@ -85,6 +87,20 @@ export default function DonationDetailsPage() {
         }
     };
 
+    /**
+     * @unreleased
+     */
+    const handleDelete = async () => {
+        try {
+            await deleteDonation(donation?.id);
+            setShowConfirmationDialog(null);
+            window.location.href = `${adminUrl}edit.php?post_type=give_forms&page=give-payment-history`;
```

Comment:
this full url would be convenient to send through the window something like: `donationsAdminUrl`

Reply:
Hey - It actually looks like this url is already being passed down from the window!
Resolved: cd6661217cbaba2e2e790e0a68c8f8c54bf765ff